src/views/Statistics.vue

```diff
@@ -83,12 +89,38 @@ export default {
         },
         { name: 'Areas climbed', value: Object.keys(areas).length },
       ];
+    },
+
+    updateItems() {
+      if (!this.userDoc || !this.indexedData || !this.indexedData.routes) {
+        return;
+      }
+
+      // If the user is on a team, retrieve the user stats from the team
+      // climbs, and also fill in team stats.
+      if (this.teamDoc && this.teamDoc.users) {
+        // Compute the stats for the user.
+        this.itemsUser = this.computeStats([this.teamDoc.users[
+          auth.currentUser.uid].climbs]);
+
+        // Compute the stats for the whole team.
+        let userClimbs = [];
+        for (const userId of Object.keys(this.teamDoc.users)) {
```

Comment:
a bit shorter:

```js
const users = this.teamDoc.users;
const userClimbs = Object.keys(users).map(uid => users[climbs]);
```

(if you use this, i'd define `users` at the top of the if block so you can use it earlier too)